db/config.cc

```diff
@@ -1314,7 +1314,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "The maximum fraction of cache memory permitted for use by index cache. Clamped to the [0.0; 1.0] range. Must be small enough to not deprive the row cache of memory, but should be big enough to fit a large fraction of the index. The default value 0.2 means that at least 80\% of cache memory is reserved for the row cache, while at most 20\% is usable by the index cache.")
     , consistent_cluster_management(this, "consistent_cluster_management", value_status::Deprecated, true, "Use RAFT for cluster management and DDL.")
     , force_gossip_topology_changes(this, "force_gossip_topology_changes", value_status::Used, false, "Force gossip-based topology operations in a fresh cluster. Only the first node in the cluster must use it. The rest will fall back to gossip-based operations anyway. This option should be used only for testing.  Note: gossip topology changes are incompatible with tablets.")
-    , recovery_leader(this, "recovery_leader", liveness::LiveUpdate, value_status::Used, "", "Host ID of the node restarted first while performing the Manual Raft-based Recovery Procedure.")
+    , recovery_leader(this, "recovery_leader", liveness::LiveUpdate, value_status::Used, "", "Host ID of the node restarted first while performing the Manual Raft-based Recovery Procedure. Warning: this option disables some guardrails for the needs of the Manual Raft-based Recovery Procedure. Make sure you unset it at the end of the procedure.")
```

Comment:
Maybe it should be a nodetool command? `nodetool recoverraftgroup0 <hostid>`.

Reply:
I'm not sure what you want this nodetool command to do. Do you want running this command to be equivalent to setting `recovery_leader`? We would have to run this command on every node, which seems a bit strange to me for a nodetool command (although I'm not a nodetool expert). We would also have to do something equivalent to unsetting `recovery_leader`, perhaps add another nodetool command? I don't see a point of doing it, it would only complicate the implementation without making the procedure simpler.

If you want `nodetool recoverraftgroup0 <hostid>` to do something else (perhaps more automation?), please be more specific.